docs/fundamentals/code-analysis/style-rules/index.md

```diff
@@ -63,19 +76,40 @@ The following table list all the code style rules by ID and options, if any.
 > | [IDE0055](formatting-rules.md) | Fix formatting | |
 > | [IDE0056](ide0056.md) | Use index operator | [csharp_style_prefer_index_operator](ide0056.md#csharp_style_prefer_index_operator) |
 > | [IDE0057](ide0057.md) | Use range operator | [csharp_style_prefer_range_operator](ide0057.md#csharp_style_prefer_range_operator) |
-> | [IDE0058](ide0058.md) | Unused expression value | [csharp_style_unused_value_expression_statement_preference](ide0058.md#csharp_style_unused_value_expression_statement_preference)<br/> [visual_basic_style_unused_value_expression_statement_preference](ide0058.md#visual_basic_style_unused_value_expression_statement_preference) |
-> | [IDE0059](ide0059.md) | Unnecessary value assignment | [csharp_style_unused_value_assignment_preference](ide0059.md#csharp_style_unused_value_assignment_preference)<br/> [visual_basic_style_unused_value_assignment_preference](ide0059.md#visual_basic_style_unused_value_assignment_preference) |
+> | [IDE0058](ide0058.md) | Remove unused expression value | [csharp_style_unused_value_expression_statement_preference](ide0058.md#csharp_style_unused_value_expression_statement_preference)<br/> [visual_basic_style_unused_value_expression_statement_preference](ide0058.md#visual_basic_style_unused_value_expression_statement_preference) |
+> | [IDE0059](ide0059.md) | Remove unnecessary value assignment | [csharp_style_unused_value_assignment_preference](ide0059.md#csharp_style_unused_value_assignment_preference)<br/> [visual_basic_style_unused_value_assignment_preference](ide0059.md#visual_basic_style_unused_value_assignment_preference) |
 > | [IDE0060](ide0060.md) | Remove unused parameter | [dotnet_code_quality_unused_parameters](ide0060.md#dotnet_code_quality_unused_parameters) |
 > | [IDE0061](ide0061.md) | Use expression body for local functions | [csharp_style_expression_bodied_local_functions](ide0061.md#csharp_style_expression_bodied_local_functions) |
 > | [IDE0062](ide0062.md) | Make local function static | [csharp_prefer_static_local_function](ide0062.md#csharp_prefer_static_local_function) |
-> | [IDE0063](ide0063.md) | Use simple 'using' statement | [csharp_prefer_simple_using_statement](ide0063.md#csharp_prefer_simple_using_statement) |
-> | [IDE0065](ide0065.md) | 'using' directive placement | [csharp_using_directive_placement](ide0065.md#csharp_using_directive_placement) |
+> | [IDE0063](ide0063.md) | Use simple `using` statement | [csharp_prefer_simple_using_statement](ide0063.md#csharp_prefer_simple_using_statement) |
+> | [IDE0064](ide0064.md) | Make struct fields writable | |
+> | [IDE0065](ide0065.md) | `using` directive placement | [csharp_using_directive_placement](ide0065.md#csharp_using_directive_placement) |
 > | [IDE0066](ide0066.md) | Use switch expression | [csharp_style_prefer_switch_expression](ide0066.md#csharp_style_prefer_switch_expression) |
+> | [IDE0070](ide0070.md) | Use <xref:System.HashCode.Combine?displayProperty=fullName> | |
```

Comment:
```suggestion
> | [IDE0070](ide0070.md) | Use <xref:System.HashCode.Combine%2A?displayProperty=fullName> | |
```

This may fix the current warning. The `%2A` (for `*`) means all overloads.

Reply:
I tend to use:

`https://xref.docs.microsoft.com/autocomplete?text=System.HashCode.Combine` to get all the valid xrefs.

You can also validate an exact xref using:

`https://xref.docs.microsoft.com/query?uid=System.HashCode.Combine` (this will show 0 results because it's not valid)

while `https://xref.docs.microsoft.com/query?uid=System.HashCode.Combine%2A` will show the expected result.

Reply:
You can also use the [Docs authoring pack](https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack) to insert valid xref links. I use this all the time:

Alt+M (to open the menu)
Choose link (I just start typing link and then select it)
Choose xref
Type in some or all of the name, e.g. "hashcode.combine"
Once the results appear, select the one you want (a specific overload or all overloads like Youssef pointed out)
Select how you want the link text to display (e.g. name with type or just name)

Reply:
Thanks both, that is very helpful!